### DISCUSSION TITLE
Button Change in UI Curtain Switch

### DISCUSSION THREAD

**Discussion by TobiasF1986** [arendst/Tasmota#12368] [Q&A] [answered]
![Screenshot_20210615-132653_Samsung Internet](https://user-images.githubusercontent.com/45529372/122045163-b13d8e80-cddd-11eb-8580-d0538a1fa9c2.jpg)

![Screenshot_20210615-045706_Samsung Internet](https://user-images.githubusercontent.com/45529372/121986361-a4e21300-cd96-11eb-8466-00df3b5edc98.jpg)
![Screenshot_20210615-045937_Samsung Internet](https://user-images.githubusercontent.com/45529372/121986364-a6134000-cd96-11eb-9fb6-1c635b8abf12.jpg)

Hello,
I flashed a 3-way curtain switch with tasmota 9.4..I took one of the curtain switch templates an I got three buttons in the ui:
1 up
2 stop
3 down. 
I would like to use SetOption80 1 but if I activate this rule, I get two buttons in the UI for up/down and a toggle 3. Now the button with the down arrow is "stop" instead of "down" . Is there a way to change the function of toggle 2 und 3 in the UI?
Thanks a lot.

**Comment by Jason2866** [comment]
Without providing the template you used, we can only guess.

**Comment by TobiasF1986** [comment]
Oh, sorry. 
Here is the template I used:
{"NAME":"Tuya Shutter","GPIO":[544,0,290,161,225,34,0,0,32,224,289,226,288,0],"FLAG":0,"BASE":18}

**Comment by TrDA-hab** [comment, marked as answer]
No, you cannot override the assignment of Relay1 and Relay2, they are "tied" to ShutterRelay1. You can only use Relay3 for your own purposes. Your example will only have this:
Relay1 up ("tied" to ShutterRelay1).
Relay2 down ("tied" to ShutterRelay1).
Relay 3 stop ("free" Relay).

More information:
https://tasmota.github.io/docs/Blinds-and-Shutters/#shutter-modes
